gensim/models/phrases.py

```diff
@@ -412,7 +411,7 @@ def load(cls, *args, **kwargs):
             if not isinstance(word, str):
                 logger.info("old version of %s loaded, upgrading %i words in memory", cls.__name__, len(model.vocab))
                 logger.info("re-save the loaded model to avoid this upgrade in the future")
-                vocab = defaultdict(int)
+                vocab = dict()
```

Comment:
More Pythonic:

```suggestion
                vocab = {}
```

(here and elsewhere)

Reply:
@piskvorky Fixed the PR+title and Descriotion. Is this acceptable? I also pushed the code to my branch after changing to `vocab = {}` but It's got getting reflected here?

Reply:
I don't understand your question. If the changed code you pushed got reflected here, that's good. Except I can't see it.

Reply:
@thalishsajeed I don't see the changes here, so it's likely you either pushed to some other branch, or not at all. Please double-check.

Reply:
@mpenkov  You can see 7 commits on my branch but only 6 commits are being reflected here. Any idea what I'd be doing wrong? 

`my develop branch`
![Capture](https://user-images.githubusercontent.com/10665298/107306416-d5453800-6aaa-11eb-9447-f9544609185d.PNG)

Reply:
Might be a Github (Microsoft) issue. I remember seeing a discussion around flaky PRs [here](https://news.ycombinator.com/item?id=26066365) (with some resolution tips too).

Reply:
Yeah, I opened a ticket with support, but haven't heard back from them.

@thalishsajeed perhaps try force-pushing?

Reply:
@mpenkov Looks like my changes are reflecting now!